mkdocs.yml

```diff
@@ -125,6 +125,7 @@ nav:
           - RClone: data_management/transfer/rclone.md
           - FileZilla: data_management/transfer/filezilla.md
       - Code Storage: data_management/code_storage.md
+      - Project Directory organization: data_management/project_directory_organization.md
```

Comment:
Before we merge this one, let's quickly put together a restructuring for Cheaha Storage (GPFS).

Make the Cheaha Storage page the index of a navigation section, then put this page under that heading.